superset-frontend/src/views/CRUD/data/dataset/AddDataset/Footer/index.tsx

```diff
@@ -104,7 +104,13 @@ function Footer({
         if (typeof response === 'number') {
           logEvent(LOG_ACTIONS_DATASET_CREATION_SUCCESS, datasetObject);
           // When a dataset is created the response we get is its ID number
-          goToPreviousUrl();
+
+          // Go to usage page if in testing mode. Will be changed
+          // to go to usage page when completed.
+          if (window.location.search === '?testing') {
+            window.location.href = `/dataset/${response}?testing`;
+          }
+          goToPreviousUrl(`/dataset/${response}?testing`);
```

Comment:
can we name this just `goToUrl` then if it's being used for more than the previous url?

Reply:
Good thinking, changed in [`this commit`](https://github.com/apache/superset/pull/22043/commits/77f5c7f40fe0c0269d5ef8a2f3e6ad2483d8e180).